Formula/tiv.rb

```diff
@@ -0,0 +1,30 @@
+class Tiv < Formula
```

Comment:
Given that the project is called `TerminalImageViewer`, I suggest we name the formula similarly. We can add an alias named `tiv` for convenience.

Reply:
So if we were to proceed with this naming, we would change both the class name and the filename?

Reply:
Yes

Reply:
So should I add the alias in this pull request?

Reply:
Done in the newest commit. I miss command-line passwords.